ddgr

```diff
@@ -1476,6 +1490,7 @@ class DdgArgumentParser(argparse.ArgumentParser):
           d keywords            new DDG search for 'keywords' with original options
                                 should be used to search omniprompt keys and indices
           x                     toggle url expansion
+          c <index>             copy url to clipboard (needs pyperclip)
```

Comment:
`index` should be within third brackets. Also, this needs to go to the man page and README.